roles/libvirt_manager/tasks/create_cloud_init_iso.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Generate cloud-init iso
+  become: false
+  vars:
+    _images_dir: "{{ [ cifmw_libvirt_manager_basedir, 'workload', ] | ansible.builtin.path_join }}"
+  block:
+    - name: "Acquire authorized_keys from hypervisor"
+      ansible.builtin.slurp:
+        src: "{{ ansible_user_dir ~ '/.ssh/authorized_keys' }}"
+      register: _authorized_keys_b64
+
+    - name: "Generate instance random UUID"
+      ansible.builtin.set_fact:
+        _instance_uuid: "{{ 99999999 | random | to_uuid | lower }}"
+
+    - name: "Render vm type based user-data"
+      vars:
+        _authorized_keys: "{{ _authorized_keys_b64.content | b64decode }}"
+        _static_ip_script: >-
```

Comment:
I think we should use network_config that cloud-init understands instead of this script? But that might be a follow up?

Reply:
I have to admit that I barely translated instructions from "old" shell instructions into cloud-init params, ignoring `network_config`. I'll have a look and try to integrate it in this PR or in a follow-up.